### DISCUSSION TITLE
Default `update_objects_updated_at` trigger

### DISCUSSION THREAD

**Discussion by moinism** [supabase/supabase#10804] [Questions] [answered]
I accidentally deleted the `update_objects_updated_at` database trigger from my account. Is there any resource I can check to re-add it?

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/storage-api/blob/master/migrations/tenant/0010-add-trigger-to-auto-update-updated_at-column.sql

I assume it was in the storage schema but did not look.

**Reply by moinism** [reply]
Thank you @GaryAustin1.
